torch/_functorch/_aot_autograd/subclass_utils.py

```diff
@@ -347,3 +347,11 @@ def compute_inner_mutated_inp_indices_from_subclass_meta(
         for i, inp in enumerate(updated_input_info)
         if inp.mutation_type == MutationType.MUTATED_OUT_GRAPH
     ]
+
+
+def subclass_setattr(s, attr, value):
+    # setattr on Subclass could fail with AttributeError e.g. NestedTensor for _min_seqlen_tensor
```

Comment:
cc @jbschlosser does this ring a bell (or sound suspicious)?

Reply:
Updated PR with removing it, avoiding to do setattr() as _min_seqlen_tensor is already contiguous.

But technically yes, it's a good design question for subclasses, if framework should do some actions on each subclass subelement indepedently or only use Subclass Authors API or __torch_dispatch__ calls on it only.

As we can shift Subclass to some inconsistent state that Authors did not expect :)

Reply:
In dynamo execution `setattr(NestedTensor(), "_min_seqlen_tenor", value)` fails.

Reply:
from offline - we realized that NJT returning `_min_seqlen_tensor` as an "inner tensor attribute" but having it secretly be a computed property that reads from a cache is a bit suspicious